RoboticonColony/Assets/Code/Market.cs

```diff
@@ -116,14 +116,14 @@ public Market Sell(ItemType item, int quantity, Inventory playerInventory)
                 //If the transfer completes without error, then the transaction is complete and a reference to this market instance is returned.
                 return this;
             }
-            catch (ArgumentOutOfRangeException)
+            catch (NotEnoughItemException)
             {
                 //If the item transfer was unsuccessful, then revert the money transfer and re-throw the exception
                 playerInventory.TransferMoney(_sellprice[item] * quantity, Stock);
                 throw;
             }
         }
-        catch(ArgumentOutOfRangeException)
+        catch(NotEnoughMoneyException)
         {
```

Comment:
I know it's too late now, but I had already changed this is my first exception pull request, are you sure you caught up with the master branch?